tests/kernel/sched/schedule_api/testcase.yaml

```diff
@@ -3,19 +3,23 @@ tests:
     filter: not CONFIG_SCHED_MULTIQ
     extra_configs:
       - CONFIG_TIMESLICING=y
+    min_ram: 36
```

Comment:
I think this is very restrictive, there are now at least ~50 platforms with 32kb or less that would be excluded based on this.

Reply:
any suggestion on how to handle this?